src/module/showList.js

```diff
@@ -0,0 +1,20 @@
+export default function populateList(obj) {
+  const displayList = document.querySelector('.displayList');
```

Comment:
- [ ] Could you please update your function to use ES6 arrow functions in line with [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)? 😊

   ![image](https://github.com/eamador1/leaderboard/assets/107952724/af44041f-9545-4a73-a165-21758813846f)